python/cudf/cudf/core/multiindex.py

```diff
@@ -397,8 +397,12 @@ def __repr__(self):
 
         if len(self) > max_seq_items:
             n = int(max_seq_items / 2) + 1
-            indices = list(range(n))
-            indices.extend(list(range(len(self) - n, len(self))))
+            indices = cudf.core.column.arange(start=0, stop=n, step=1)
+            indices = indices.append(
+                cudf.core.column.arange(
+                    start=len(self) - n, stop=len(self), step=1
+                )
+            )
```

Comment:
Any ideas on if we could generate it in one go instead of two `arange` calls + an `append` call?

Reply:
If you are okay with this approach, we could utilize `cupy.r_` here:

```python
>>> cp.r_[cp.arange(10, 12), cp.arange(20, 22)]
array([10, 11, 20, 21])
```

Reply:
This will still have to go through two `arange` calls because `slice` is yet to be supported in `cp.r_`: https://github.com/cupy/cupy/blob/7cc68ea831f82e463f8c6c36cc0a8fd5ae6387a4/cupy/_indexing/generate.py#L54

Reply:
Let's add a `#TODO` for now and maybe there's a future cudf enhancement where `arange` can take a vector of start and stop pairs instead of just a single one.

Reply:
Added a `#TODO`